<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>clsMainTestWindow</class>
 <widget class="QMainWindow" name="clsMainTestWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1475</width>
    <height>639</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>WKE LightWave</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../images/res.qrc">
    <normaloff>:/lightWave.png</normaloff>:/lightWave.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">clsMainTestWindow
{
	background-color:rgb(150,150,251);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>2</number>
    </property>
    <property name="leftMargin">
     <number>2</number>
    </property>
    <property name="topMargin">
     <number>2</number>
    </property>
    <property name="rightMargin">
     <number>2</number>
    </property>
    <property name="bottomMargin">
     <number>2</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="btnNewSetup">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>楷体</family>
          <pointsize>13</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>新建
通道</string>
        </property>
        <property name="icon">
         <iconset resource="../images/res.qrc">
          <normaloff>:/new.png</normaloff>:/new.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>26</width>
          <height>26</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnOpenSetup">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>楷体</family>
          <pointsize>13</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>打开
步骤</string>
        </property>
        <property name="icon">
         <iconset resource="../images/res.qrc">
          <normaloff>:/open.png</normaloff>:/open.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>26</width>
          <height>26</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnSaveSetup">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>楷体</family>
          <pointsize>13</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>保存
步骤</string>
        </property>
        <property name="icon">
         <iconset resource="../images/res.qrc">
          <normaloff>:/stepSave.png</normaloff>:/stepSave.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>26</width>
          <height>26</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnTrig">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>楷体</family>
          <pointsize>13</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>手动
触发</string>
        </property>
        <property name="icon">
         <iconset resource="../images/res.qrc">
          <normaloff>:/single.png</normaloff>:/single.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>26</width>
          <height>26</height>
         </size>
        </property>
        <property name="shortcut">
         <string>Space</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnReptive">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>楷体</family>
          <pointsize>13</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>重复
测试</string>
        </property>
        <property name="icon">
         <iconset resource="../images/res.qrc">
          <normaloff>:/Auto.png</normaloff>:/Auto.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>26</width>
          <height>26</height>
         </size>
        </property>
        <property name="shortcut">
         <string>Return</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnRunning">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>楷体</family>
          <pointsize>13</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>编辑
模式</string>
        </property>
        <property name="icon">
         <iconset resource="../images/res.qrc">
          <normaloff>:/settings3.png</normaloff>:/settings3.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>26</width>
          <height>26</height>
         </size>
        </property>
        <property name="shortcut">
         <string>Ctrl+R</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <property name="autoExclusive">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnSettings">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>楷体</family>
          <pointsize>13</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>运行
模式</string>
        </property>
        <property name="icon">
         <iconset resource="../images/res.qrc">
          <normaloff>:/Start1.png</normaloff>:/Start1.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>26</width>
          <height>26</height>
         </size>
        </property>
        <property name="shortcut">
         <string>Ctrl+E</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <property name="autoExclusive">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnChannelCalibration">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>楷体</family>
          <pointsize>13</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>通道
校准</string>
        </property>
        <property name="icon">
         <iconset resource="../images/res.qrc">
          <normaloff>:/settings.png</normaloff>:/settings.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>26</width>
          <height>26</height>
         </size>
        </property>
        <property name="shortcut">
         <string>Return</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnRunSettings">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>楷体</family>
          <pointsize>13</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>运行
设置</string>
        </property>
        <property name="icon">
         <iconset resource="../images/res.qrc">
          <normaloff>:/Settings2.png</normaloff>:/Settings2.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>26</width>
          <height>26</height>
         </size>
        </property>
        <property name="shortcut">
         <string>Return</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnAboutMe">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>楷体</family>
          <pointsize>13</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>软件
信息</string>
        </property>
        <property name="icon">
         <iconset resource="../images/res.qrc">
          <normaloff>:/lightWave.png</normaloff>:/lightWave.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>26</width>
          <height>26</height>
         </size>
        </property>
        <property name="shortcut">
         <string>Return</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnSignal">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>楷体</family>
          <pointsize>13</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>显示
信号</string>
        </property>
        <property name="icon">
         <iconset resource="../images/res.qrc">
          <normaloff>:/manualTrig.png</normaloff>:/manualTrig.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>26</width>
          <height>26</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="stepWidget" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QScrollArea" name="scrollArea">
           <property name="lineWidth">
            <number>0</number>
           </property>
           <property name="widgetResizable">
            <bool>true</bool>
           </property>
           <widget class="QWidget" name="scrollAreaWidgetContents">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>1302</width>
              <height>529</height>
             </rect>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QWidget" name="itemWidget" native="true">
               <property name="styleSheet">
                <string notr="true">background-color: rgb(203, 215, 255);
</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="wgtSignalPannel" native="true">
        <property name="minimumSize">
         <size>
          <width>165</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>165</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1475</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="myStatusBar">
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>11</pointsize>
    </font>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../images/res.qrc"/>
  <include location="../images/res.qrc"/>
  <include location="../images/res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btnRunning</sender>
   <signal>toggled(bool)</signal>
   <receiver>btnTrig</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1127</x>
     <y>74</y>
    </hint>
    <hint type="destinationlabel">
     <x>340</x>
     <y>75</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnSettings</sender>
   <signal>toggled(bool)</signal>
   <receiver>btnNewSetup</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1213</x>
     <y>75</y>
    </hint>
    <hint type="destinationlabel">
     <x>69</x>
     <y>57</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnSettings</sender>
   <signal>toggled(bool)</signal>
   <receiver>btnSaveSetup</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1213</x>
     <y>68</y>
    </hint>
    <hint type="destinationlabel">
     <x>225</x>
     <y>43</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnSettings</sender>
   <signal>toggled(bool)</signal>
   <receiver>btnOpenSetup</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1213</x>
     <y>75</y>
    </hint>
    <hint type="destinationlabel">
     <x>147</x>
     <y>40</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnSignal</sender>
   <signal>clicked(bool)</signal>
   <receiver>wgtSignalPannel</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1471</x>
     <y>75</y>
    </hint>
    <hint type="destinationlabel">
     <x>1471</x>
     <y>260</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnRunning</sender>
   <signal>toggled(bool)</signal>
   <receiver>btnReptive</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1127</x>
     <y>69</y>
    </hint>
    <hint type="destinationlabel">
     <x>426</x>
     <y>75</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnRunning</sender>
   <signal>toggled(bool)</signal>
   <receiver>btnChannelCalibration</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1087</x>
     <y>50</y>
    </hint>
    <hint type="destinationlabel">
     <x>1275</x>
     <y>54</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnRunning</sender>
   <signal>toggled(bool)</signal>
   <receiver>btnRunSettings</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1078</x>
     <y>40</y>
    </hint>
    <hint type="destinationlabel">
     <x>1367</x>
     <y>56</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
